docs/news/news-v5.0.x.rst

```diff
@@ -4,14 +4,20 @@ Open MPI v5.0.x series
 This file contains all the NEWS updates for the Open MPI v5.0.x
 series, in reverse chronological order.
 
-Open MPI version v5.0.2rc1
+Open MPI version v5.0.2rc2
 --------------------------
-:Date: 24 January 2024
+:Date: 31 January 2024
 
 - Internal PMIx and PRRTe versions:
 
-  - PMIx (v4.2.8-3-ge68bdfcf). Commit hash: ``e68bdfcf4133cb4c1af31c591bb1d26ed63fe911``.
-  - PRRTE (v3.0.3-5-g8bd181af08). Commit hash: ``8bd181af0836375e66c741c1c94364d8a44918e5``.
+  - PMIx (v4.2.8). Commit hash: ``d1c1ed0c2e64f19ad49291241a43630ea7fdce28``.
+  - PRRTE (v3.0.3). Commit hash: ``e383f5ad70c2633420d3425e9fb67e69b6bfd9c4``.
+
+  .. note:: 5.0.2rc2 has pinned PMIx and PRRTe to the respective previous
+            release tags. This effectively excludes the submodule updates
+            in rc1. This action is needed in order to ensure the internal
+            modules always refer to stable releases that can be matched by
+            external installations.
```

Comment:
This is not word wrapped well here in the github UI, but I suggest text something like the following:

```suggestion
  .. note:: 5.0.2rc2 has pinned PMIx and PRRTe to the respective previous
            release tags (...list the actual release tags here...). This effectively excludes the PMIx and PRRTE submodule updates
            that were included in rc1. We are specifically choosing to do this because the Open MPI community
            needs a v5.0.2 release before we can do full testing with the upcoming PMIx v4.2.WHATEVER release.  We expect to have an Open MPI v5.0.3 release in the near future with the upcoming PRRTE vWHATEVER and PMIX vWHATEVER releases.
```

Reply:
Also, I think you need to include a list of known issues that have been fixed since the previous PMIX/PRRTE release tags that will now not be included in v5.0.2[rcX].

Reply:
Please see update. I want to be concise with this paragraph, so I picked out the most relevant change in pmix/prrte which is the pset membership query bugfix from Howard.

I disagree on promising a specific release that will include new prrte/pmix - the release might happen for a different reason. We will evaluate case by case.